microsoft-edge/web-platform/release-notes/142.md

```diff
@@ -0,0 +1,486 @@
+
+* [Explainer](https://github.com/webmachinelearning/prompt-api/blob/main/README.md)
+* [Feedback](https://github.com/webmachinelearning/prompt-api/issues)
+* [Register](https://developer.microsoft.com/microsoft-edge/origin-trials/trials/b7d35247-b855-4b08-b237-89e7a5056117)
+
+See also:
+* [Prompt a built-in language model with the Prompt API](../prompt-api.md)
+
+
+<!-- ---------- -->
+###### Incoming Call Notifications
+
+Expires on May 19, 2026.
+
+Extend the Notifications API to allow installed PWAs to send incoming call notifications -- that is, notifications that have call-styled buttons and a ringtone.  This extension helps VoIP web apps create more engaging experiences by making it easier for users to easily recognize a calling notification and answer it.  This feature also helps bridge the gap between native and web implementations of apps that have them both.
+
+* [Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/Notifications/notifications_actions_customization.md)
+* [Feedback](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues)
+* [Register](https://developer.chrome.com/origintrials/#/register_trial/2876111312029483009)
+
+See also:
+* [Request for feedback: Incoming call notifications API for web apps](https://blogs.windows.com/msedgedev/2025/02/04/request-for-feedback-incoming-call-notifications-api/)
+
+
+<!-- ---------- -->
+###### Full frame rate render blocking attribute
+
+Expires on March 24, 2026
+
+The new `full-frame-rate` render blocking attribute instructs the browser to render the page at a lower frame rate to reserve resources while parsing critical content.  After the critical content has been parsed, the browser restores its normal frame rate.
+
+For example `<link rel="expect" href="#critical-content" blocking="full-frame-rate">` lowers the frame rate of the browser until the `#critical-content` element is parsed.
+
+The `full-frame-rate` render blocking attribute is informational only.  The browser may decide to lower the frame rate before parsing a blocking element, such as at the very beginning of the loading phase.  The browser may also decide to restore the frame rate before the blocking element list becomes empty, such as after a timeout, or after certain user interactions.
+
+* [Explainer](https://github.com/whatwg/html/issues/11070)
+* [Feedback](https://github.com/whatwg/html/issues/11070)
+* [Register](https://developer.microsoft.com/microsoft-edge/origin-trials/trials/e43c41e4-b9d3-4128-a89f-a2e4d0de80f4)
+
+
+<!-- ------------------------------ -->
+#### Microsoft Edge-only origin trials
```

Comment:
should Edge heading specify new or active, like Chromium heading?
@captainbrosset